deploy.sh

```
@@ -0,0 +1,327 @@
+    docker save "$BACKEND_IMAGE" | ssh -i "$SSH_KEY" "$SSH_USER@$SSH_HOST" "docker load"
+    docker save "$FRONTEND_IMAGE" | ssh -i "$SSH_KEY" "$SSH_USER@$SSH_HOST" "docker load"
+    
+    print_info "Images transferred successfully"
+fi
+
+# Deploy with Docker Compose
+print_info "Deploying application..."
+
+# Pull images if not built locally
+if [ "$BUILD_LOCAL" != true ]; then
+    ssh -i "$SSH_KEY" "$SSH_USER@$SSH_HOST" << EOF
```

Shellcheck (0.11.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)